src/mo_config/flags.ml

```diff
@@ -44,7 +44,7 @@ let profile_field_names : string list ref = ref []
 let public_metadata_names : string list ref = ref []
 let omit_metadata_names : string list ref = ref []
 let compiled = ref false
-let error_detail = ref 2
+let error_detail = ref 4
```

Comment:
BTW, are you sure that we want to change default?  Then we have to update `src/mo_config/args.ml` 

I wouldn't do it, at least without any arguments